packages/code-du-travail-frontend/src/outils/IndemniteLicenciement/components/AbsencePeriods.js

```diff
@@ -39,25 +39,25 @@ function AbsencePeriods({ name, onChange }) {
                   subscribe={{ error: true, touched: true }}
                   render={({ input, meta: { touched, error, invalid } }) => (
                     <>
-                      <Input {...input} size="7" invalid={touched && invalid} />
-                      <DelButton
-                        variant="link"
-                        type="button"
-                        onClick={() => fields.remove(index)}
-                      >
-                        Supprimer
-                      </DelButton>
-                      {error && touched && invalid ? (
-                        <InlineError>{error}</InlineError>
-                      ) : null}
+                      <Input
+                        {...input}
+                        size="7"
+                        type="number"
+                        invalid={touched && invalid}
+                      />
+                      {error && touched && invalid && (
+                        <BlockError>{error}</BlockError>
+                      )}
                     </>
                   )}
                 />
               </DurationCell>
+              <DelButton type="button" onClick={() => fields.remove(index)}>
```

Comment:
missing variant="link" 
also it would be better to group the Input and the button inside the same container to avoid a layout jump when the error message shows up

Reply:
Already checked, there is no jump this way !

Reply:
I just tried it 
![image](https://user-images.githubusercontent.com/160320/61289290-4c302700-a7c9-11e9-94aa-061a7a93063a.png)
![image](https://user-images.githubusercontent.com/160320/61289307-581be900-a7c9-11e9-9b8e-6f160a7dc999.png)

Reply:
My bad, I got confused ! I'll group it in the same container nice catch !